### ISSUE TITLE
Error: "Submit" button in multi-step form reloads the page.

### ISSUE THREAD

**Issue by TigranSaakyan** [elementor/elementor#11677]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description

When I am using a single-step form, the "submit" button works just fine. However, when I divide the form into several steps - the submit button at the end of the form reloads the page instead of submitting it.  All fields get emptied and I return to the start of the form. After the reload it also brings me to the top of the page.

When I try to exit or reload the page, google chrome gives a notification "Confirm Form Resubmission" as if the information is still inside the form (but fields are actually empty).

## Steps to reproduce

Thanks to [kbrebel04](https://github.com/kbrebel04), the issue is found to occur only when the "step" setting of multistep form is set to "Progress bar" or "None". [kbrebel04](https://github.com/kbrebel04) describes good steps to reproduce this issue in comments below. 

To reproduce this issue, create a long multi-step form (it's kinda specific to my situation, but hope it helps when you try to solve it)

Step 1:
2 regular text fields
1 field for email
1 field for tel

Step 2:
8 regular text fields
1 checkbox field

Step 3:
2 regular text fields

reCAPTCHA v.2 (i tried disabling it- it has no effect on the form submission outcome)

I am using Astra theme
Elementor Version 2.9.12
Elementor Pro Version 2.10.2
I disabled all JS/CSS optimization plugins

## Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by asimcard** [comment]
I may have had this too. Will keep an eye out for it

**Comment by NereuFajardo** [comment]
I have the same problem. Following

**Comment by kbrebel04** [comment]
Same issue. I have created a new post on how to replicate this exactly here: https://github.com/elementor/elementor/issues/11697

TL;DR: The issue is related to the multistep form and only happens with "progress bar" or "none" selected for the STEP TYPE. 

Merge please.

**Comment by TigranSaakyan** [comment]
If you don't mind, I'l add that detail to this post as well. "Progress bar/None" setting is actually the reason it glitches. Thank you

**Comment by Worldwebscapes** [comment]
Same bug here. Updated to Free 2.9.13 Pro 2.10.2. Forms only work if default steps are used.

**Comment by TigranSaakyan** [comment]
I also noticed that multi-step forms appear as regular (single-step) forms on mobile and tablet

**Comment by arielk** [comment]
This issue has been resolved in Elementor v2.10.3

Feel free to update

Thanks!
